plyer/platforms/linux/orientation.py

```diff
@@ -0,0 +1,27 @@
+from subprocess import call, check_output
+from plyer.facades import Orientation
+from plyer.utils import whereis_exe
+
+
+class XrandrOrientation(Orientation):
+
+    def _set_landscape(self, **kwargs):
+        self.rotate = 'normal'
+        self.screen = check_output("xrandr -q | grep ' connected' \
+                              |  head -n 1 | cut -d ' ' -f1", shell=True)
```

Comment:
@KeyWeeUsr Done both changes you suggested. Now please check the PR and merge this.